packages/ui/core-components/src/lib/atoms/inputs/slider/Slider.svelte

```diff
@@ -25,35 +26,11 @@
 	/** @type {number|undefined} */
```

Comment:
Since `min` has a default value (`0`), it will never be `undefined`, so we can remove `undefined` from the type

```suggestion
	/** @type {number} */
```